gui/options.cpp

```diff
@@ -1997,6 +2001,7 @@ void GlobalOptionsDialog::build() {
 	// Set _savePath to the current save path
 	Common::String savePath(ConfMan.get("savepath", _domain));
 	Common::String themePath(ConfMan.get("themepath", _domain));
+	Common::String iconPath(ConfMan.get("iconpath", _domain));
```

Comment:
Although it was not yet exposed in the GUI, we are already using "iconspath" (with a 's') in the taskbar: https://github.com/scummvm/scummvm/blob/master/common/taskbar.h#L164

We should probably us the same here.

Reply:
I have replaced "iconpath" with the "iconspath" config key in 2c06ecf.